### DISCUSSION TITLE
What does block the packing on Windows ?

### DISCUSSION THREAD

**Discussion by Starmania** [oclif/oclif#1630] [General]
This line https://github.com/oclif/oclif/blob/13ddf25419936f359bd22fd72c19ea1e607178f6/src/commands/pack/tarballs.ts#L26 block the packing on Windows. But why ?

The reason I'm asking this is to know it it would be possible to pack on a platform like MSYS2.

**Comment by mdonnalley** [comment]
See https://github.com/oclif/oclif/issues/1593
